tools/pythonpkg/src/include/duckdb_python/arrow_array_stream.hpp

```diff
@@ -46,3 +72,16 @@ class PythonTableArrowArrayStreamFactory {
 	                                 ArrowStreamParameters &parameters, ClientConfig &config);
 };
 } // namespace duckdb
+
+namespace pybind11 {
+namespace detail {
+template <>
+struct handle_type_name<duckdb::pyarrow::RecordBatchReader> {
+	static constexpr auto name = _("pyarrow.lib.RecordBatchReader");
+};
+template <>
+struct handle_type_name<duckdb::pyarrow::Table> {
+	static constexpr auto name = _("pyarrow.lib.Table");
```

Comment:
Constexpr should be uppercase

Reply:
This name is from pybind11, I can't change it